src/main/java/org/apereo/portlet/soffit/model/v1_0/Context.java

```diff
@@ -22,19 +22,26 @@
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.HashSet;
+import java.util.List;
 import java.util.Map;
 import java.util.Set;
 
+/**
+ * Provides information about the portal or container in which the soffit is
+ * running.
+ *
+ * @author drewwills
+ */
 public class Context {
 
     private String portalInfo;
-    private String serverInfo;
 
-    private Map<String,String> initParameters = new HashMap<>();
-    private Set<String> containerRuntimeOptions = new HashSet<>();
-    private Set<String> supportedModes = new HashSet<>();
```

Comment:
These items -- initParameters, containerRuntimeOptions, & supportedModes -- are potentially significant to the interaction between the JSR-268 container & the SoffitConnectorController;  but they probably aren't significant to the interaction between the SoffitConnectorController & the remote soffit.  In any case, all connectors share a `<portlet>` definition in portlet.xml, so all 3 of these would always send the same information.